.depcheckrc.yml

```diff
@@ -49,15 +49,11 @@ ignores:
   # storybook
   - '@storybook/cli'
   - '@storybook/core'
-  - '@storybook/addon-designs'
```

Comment:
Not used and removed all instances 

<img width="371" height="259" alt="Screenshot 2025-08-11 at 9 36 26 AM" src="https://github.com/user-attachments/assets/0ce12d86-98c8-4fc3-9578-86a0e81a90bf" />